lte/gateway/python/magma/enodebd/enodeb_status.py

```diff
@@ -49,23 +49,21 @@
 _gps_lat_cached = None
 _gps_lon_cached = None
 
-EnodebStatus = NamedTuple(
-    'EnodebStatus',
-    [
-        ('enodeb_configured', bool),
-        ('gps_latitude', str),
-        ('gps_longitude', str),
-        ('enodeb_connected', bool),
-        ('opstate_enabled', bool),
-        ('rf_tx_on', bool),
-        ('rf_tx_desired', bool),
-        ('gps_connected', bool),
-        ('ptp_connected', bool),
-        ('mme_connected', bool),
-        ('fsm_state', str),
-        ('cell_id', int),
-    ],
-)
+
+class EnodebStatus(NamedTuple):
+    enodeb_configured: bool
+    gps_latitude: str
+    gps_longitude: str
+    enodeb_connected: bool
+    opstate_enabled: bool
+    rf_tx_on: bool
+    rf_tx_desired: bool
+    gps_connected: bool
+    ptp_connected: bool
+    mme_connected: bool
+    fsm_state: str
+    cell_id: int
+
 
 # TODO: Remove after checkins support multiple eNB status
 MagmaOldEnodebdStatus = namedtuple(
```

Comment:
@andreilee Do you think this can be removed now? Asking because you seemed to have added the TODO :D

Reply:
@mpfirrmann If unclear, if this can be removed, but you see it out of scope, we can also create a follow up GH issue, such that it will not be forgotten.
Thoughts?

Reply:
Not necessarily out of scope, but waiting for feedback from @andreilee before deleting the code.

Reply:
@themarwhal, @andreilee, @Neudrino
After taking a look myself: removing the `MagmaOldEnodebdStatus` seems out of scope for this PR. It would involve changes at several `enodeb` files; enough for a seperate PR which I could create if the code should be removed.